test/lit/wasm-split/call_exports.mjs

```diff
@@ -0,0 +1,25 @@
+// Instantiates an instrumented module, calls the given exports, then collects
+// its wasm-split profile and writes it to a given file.
+//
+// Usage:
+//
+//  node call_exports.mjs <module> <profile> <export>*
```

Comment:
what does this lit test do? I don't see any checks for comparing anything... maybe I just don't know enough about lit.

Reply:
This is not a lit test, this is a script invoked by profile-guided.wast (which _is_ a lit test).